Firmware/lcd.cpp

```diff
@@ -341,7 +343,6 @@ static void FORCE_INLINE lcd_set_current_row(uint8_t row)
 /// @return row offset which the LCD register understands
 static uint8_t __attribute__((noinline)) lcd_get_row_offset(uint8_t row)
 {
-	uint8_t row_offsets[] = { 0x00, 0x40, 0x14, 0x54 };
 	return row_offsets[min(row, LCD_HEIGHT - 1)];
```

Comment:
I'm pretty sure you must use pgm_read_byte for the progmem array

Reply:
You're right, I forgot about that part.

Reply:
@leptun Fixed. Flash usage increased by 4 bytes. I'll run this again quickly on my machine later this evening.